core/controllers/ingest.py

```diff
@@ -170,25 +167,24 @@ def extract_process_validate_tables(
                     f"{config['extract']['worksheet_name']} {error.cell.str_ref if error.cell else ''}:"
                     f" {error.message}"
                 )
+        tables[table_name] = table.df
```

Comment:
We'll still need to keep the `Table`s for now because cross-table validation will use them

Reply:
Just had a thought - makes more sense for you to just make this change in your cross-table validation PR. All I'd be doing here would be to assign the `Table` not the `DataFrame` to the `tables` dict, but then to keep the interface of `extract_process_validate_tables` the same (`DataFrame`s returned), in the next line of code I'd need to add something like `tables = {table_name: table.df for table_name, table in tables.items()}` anyway